db/migrate/201904101430_fix_mtu_min_value.rb

```diff
@@ -0,0 +1,8 @@
+class FixMTUMinimumValue < ActiveRecord::Migration[5.2]
+  def up
+    Subnet.where("mtu < 68 AND mtu IS NOT NULL").each do |s|
```

Comment:
There is already a constraint in the DB ensuring mtu can't be null, so no need to check for that. 
You can also use `update_all` to directly update the value in the DB directly instead of loading all subnets for updating them:
```suggestion
    Subnet.where("mtu < 68").update_all(mtu: 68)
```